css/style.css

```diff
@@ -95,6 +95,20 @@ div.recently-dropped-toggle-container {
 }
 
 /*
+ * ban list
+ */
+
+.ban-source-toggle-container {
+  text-align: right;
+  width: 116px;
+}
+
+.ban-source-toggle-container > ul {
+  text-align: left;
+  width: 150px;
+}
```

Comment:
It's finicky to use specific pixel dimensions on a small text element like this. If anything small changes like we decide to change the body's font-size or the client can't fetch the right font or some browser that we haven't tested just renders text a little bit differently, this could go back to being unaligned or start wrapping to two lines and I probably wouldn't catch it.

I don't know of a way to get this effect cleanly, but I'd rather have it left-aligned and move a bit than right aligned and prone to breaking.